.github/ISSUE_TEMPLATE/feature_request.yml

```
@@ -0,0 +1,33 @@
+name: Feature request
+description: Got a cool idea you would like implemented? Share it with us!
+labels: ["enhancement"]
+
+body:
+  - type: textarea
+    id: summary
+    attributes:
+      label: Summary
+      description: Small summary of the feature.
+    validations:
+      required: true
+
+  - type: textarea
+    id: problem
+    attributes:
+      label: Problem
+      description: What problem will this feature solve (if any)?
```

This section could probably be changed to something like "Why is this needed?"
```suggestion
  - type: textarea
    id: problem
    attributes:
      label: Why is this needed?
      description: Why should this feature be implemented? What problem(s) would it solve?
```

Additionally, should this be marked as required?

---

I think in the new context, as it's now asking why the feature is needed rather than asking if it solves any problems, I think it should be. That way contributors can justify their feature before anyone has to enquire about it.

---

Ok, then required it is 👍